src/main/java/seedu/address/logic/commands/EditCommand.java

```diff
@@ -128,7 +137,9 @@ public static class EditPersonDescriptor {
         private Name name;
         private Phone phone;
         private Email email;
-        private Address address;
+        private Telegram telegram;
+        private Slack slack;
```

Comment:
As we discussed before, we should use a hashMap to store this contact information such as `HashMap<ContactEnum, Contact>`